grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -73,8 +73,13 @@ class OrderController {
 
         def orders = orderService.getOrders(orderTemplate, statusStartDate, statusEndDate, params)
 
-        if (params.format && orders) {
+        def ordersDerivedStatus
```

Comment:
In the future, you can just call something like this derivedOrderStatuses. Otherwise it's awkward to read.